package.json

```diff
@@ -111,6 +111,7 @@
     "jest": "^23.5.0",
     "lint-staged": "^7.2.2",
     "npm-run-all": "^4.1.3",
+    "prettier": "^1.14.2",
```

Comment:
Thanks for adding this! It missing was simply an oversight 😅

Reply:
That's what I thought! 👌